app/models/form/sales/questions/shared_ownership_type.rb

```diff
@@ -0,0 +1,22 @@
+class Form::Sales::Questions::SharedOwnershipType < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "type"
+    @check_answer_label = "Type of shared ownership sale"
+    @header = "What is the type of shared ownership sale?"
+    @hint_text = "A shared ownership sale is when the purchaser buys up to 75% of the property value and pays rent ro the Private Registered Provider (PRP) on the remaining portion"
```

Comment:
Should this say "to the Private..."?

Reply:
Yup, I just copied it over from the ticket 😅 Good spot!